csharp/src/Ice/Connection.cs

```diff
@@ -145,7 +145,7 @@ public TimeSpan IdleTimeout
         /// </param>
         /// <returns>A proxy that matches the given identity and uses this connection.</returns>
         public T CreateProxy<T>(Identity identity, ProxyFactory<T> factory) where T : class, IObjectPrx =>
-            factory(new Reference(_communicator, this, identity));
+            factory(new Reference(_communicator, this, identity, invocationInterceptors: null));
```

Comment:
I find this odd. We're using a convenience Reference ctor with just the minimum number of parameters. Why add invocationInterceptors to this ctor?